lib/alephant/broker/request/batch.rb

```diff
@@ -10,17 +10,17 @@ class Batch
         attr_reader :batch_id, :components, :load_strategy
 
         def initialize(component_factory, env)
-          logger.info "Request::Batch#initialize: id: #{env.data['batch_id']}"
+          logger.info "Request::Batch#initialize: id: #{env.data['batch_id']}" if env.data
```

Comment:
This is fine in here for now, but when this hits production it might be a good idea to remove this as extra logging can have a dramatic effect on performance/throughput.  What's the log level on pre-live/live?  If it's WARN or ERROR we're ok, if not we'll have to look at this.